### DISCUSSION TITLE
f4 T13 release issue/question

### DISCUSSION THREAD

**Discussion by thunderpipesusa** [mriscoc/Ender3V2S1#623] [Q&A]
So I have given up on trying to compile something that contains UBL or LA and for the T13 Thermistor my S1 Pro has.  I have installed the current T13 release 20221222

When I go to the tramming wizard, it is not tramming in equal distances from the edges.... the area trammed is closer to the front of the bed and closer to the left side of bed, not centered in the work plane.  When I go to advanced under mesh leveling, the mesh inset is set to x min of 20, x max of 198.2, y min of 20, and y max of 179.5.  If I adjust any of that to build a mesh that is equidistant from the edges of my 220 x 220 bed.... when running the tramming wizard, it only probes one spot (the front left) and then tells me to lower that point, it never goes to the other locations. 

There are 2 options in the inset menu, maximize area and center area, maximize area only adjusts the above values from 20 to 10 on the mins.  The center area feature does center but shrinks the area probed to 40.5x179.5

I didnt have this issue with the 20221022 version of the T13 Experimental.

Any advice is appreciated.

**Comment by mriscoc** [comment]
Please flash the 20221022 version and test again, I think that you forgot to set the XY MIN/MAX positions.

**Reply by thunderpipesusa** [reply]
I have not customized the fan or the bed on this s1 pro, should be just like the s1, and the calibration guide says that the default sizes are in the firmware already.  it states to find min and max if the printer has been modified which it has not.  I dont have a current copy of the 20221022 version on my pc and am now trying to find min max per the calibration guide.  i will update when complete

**Reply by thunderpipesusa** [reply]
yea im at a loss now.  I set x min to -10 and y min to -7
the max i set to 227 for x and y
the home works and puts the nozzle at 110,110

when i go to tramming wizard i am still inset quite a bit 
the mesh inset centered is at 33.5x186.5

am i not able to mesh the full 220x220 printable area?
If i adjust the mesh inset to 10x210.... the tramming wizard again only probes the front left corner and then is done

the maximize area is showing 10x195.2 for 'x' and 10x186.5 for 'y'
why isnt the maximize area equal as this bed is a square 220x220 (234x234 actual)

**Reply by mriscoc** [reply]
It is near to imposible for the CRtouch to reach all the printable area, because of the physical limits and **probe offsets**. You can test that by disabling motors and moving the print head with your hand trying to reach the right and back sides of your bed with the tip of the CRTouch.

**Reply by thunderpipesusa** [reply]
and that my friend makes perfect sense! thanks so much for your time.  I wish I could find a new F1 Board to place in this thing to utilize more features lol Have a great day
